doc/sphinx/source/recipes/recipe_hyint.rst

```diff
@@ -0,0 +1,128 @@
+
+The hydroclimatic indices calculated by the diagnostic and included in the output are defined as follows:
+
+* PRY = mean annual precipitation
+* INT = mean annual precipitation intensity (intensity during wet days, or simple precipitation intensity index SDII)
+* WSL = mean annual wet spell length (number of consecutive days during each wet spell)
+* DSL = mean annual dry spell lenght (number of consecutive days during each dry spell)
+* PA  = precipitation area (area over which of any given day precipitation occurs)
+* R95 = heavy precipitation index (percent of total precipitation above the 95% percentile of the reference distribution)
+* HY-INT = hydroclimatic intensity. HY-INT = normalized(INT) x normalized(DSL).
+ 
+ 
+ 
+Available recipes and diagnostics
+---------------------------------
+ 
+Recipes are stored in recipes/
+ 
+* recipe_hyint.yml (evaluating the 6 hydroclimatic indices) 
+* recipe_hyintex.yml (also including a selection of ETCCDI indices)
+ 
+Diagnostics are stored in diag_scripts/hyint/
+ 
+* hyint.R
+ 
+and subroutines
+
+* hyint_diagnostic.R
+* hyint_functions.R
+* hyint_parameters.R
+* hyint_plot_trends.R
+* hyint_etccdi_preproc.R
+* hyint_metadata.R
+* hyint_plot_maps.R
+* hyint_preproc.R
+* hyint_trends.R
+
+ 
+User settings
+-------------
+ 
+*Required settings for script*
+
+
+* norm_years: first and last year of reference normalization period to be used for normalized indices
+
+* selfields: indices to be analysed and plotted. Select one or more fields from the following list (order-sensitive) as a numerical index: 1=SDII, 2=DSL, 3=WSL, 4=HY-INT, 5=ABS_INT, 6=ABS_DSL, 7=ABS_WSL, 8=PA, 9=R95
+
+* selregions: Select regions for timeseries and maps from the following list as a numerical index: 1=World, 2=World60 (60S/60N), 3=Tropics (30S/30N), 4=South 5=America, 6=Africa, 7=North America, 8=India, 9=Europe, 10=East-Asia, 11=Australia
```

Comment:
There is a mismatch between the documentation here and in the recipe. When I try to run the recipe, region 11 does not appear to exist(?). I think it would be much more user friendly and reliable to use names (i.e. strings) for identifying regions instead of these numbers.

Reply:
Typo in the numbering written in documentation, changed.

Reply:
Done